SU-SOE/stanford_soe_helper
# 7.x 2.x spotlight pages

#  READY FOR REVIEW

# Summary

- Adds related page to manage content
- Adds Related Page field to Spotlight content type

For both Horizontal and 2 span6 views:
- Adds "Any" block displays which use entity reference to display related spotlight nodes on landing pages and Stanford pages.
- Adds "alumni" block displays
- Adds related department contextual filter

- Adds all alumni spotlight page. 
- Fixes all students and faculty pages with correct blocks

# Needed By (Date)
- yesterday

# Criticality
- yeah

# Steps to Test

- Pull latest
- Revert 3 features
- Place Horizontal & 2 span6 blocks to test
- Alumni, related department, related page, button and no button.

- Navigate and verify appropriate behavior, page, and blocks at:
/spotlight
/spotlight/faculty
/spotlight/students
/spotlight/alumni

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)

https://stanfordits.atlassian.net/browse/SOE-2127
https://stanfordits.atlassian.net/browse/SOE-2043
https://stanfordits.atlassian.net/browse/SOE-2085

Evaluate PR:
https://stanfordits.atlassian.net/browse/SOE-2132

Dev: https://stanfordits.atlassian.net/browse/SOE-2106
PR: https://stanfordits.atlassian.net/browse/SOE-2135

## Related PRs
Incorporates https://github.com/SU-SOE/stanford_soe_helper/pull/88

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
